spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/pulsar/PulsarProperties.java

```diff
@@ -801,6 +802,11 @@ public static class Listener {
 		 */
 		private SchemaType schemaType;
 
+		/**
+		 * Number of threads used by listener container.
```

Comment:
It may be helpful to elaborate on this one just a bit, maybe something like:
> Number of listener threads used by the container to process messages concurrently

Wdyt?

Reply:
>  tried to aligned property description with these

Yep, I saw those descriptions as well. I think there is room for improvement in the spring-kafka properties. This is really a nit and consistency is a good thing. As you said, let's see what Boot peepz think.